### PR TITLE
Remove sidekiq

### PR DESCRIPTION
**Pull request by chcholman**
@galacto Hey! Would you take a look at this and make sure things look okay?

### CODE REVIEW

File: config/initializers/carrierwave_backgrounder.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 CarrierWave::Backgrounder.configure do |c|
   #c.backend :delayed_job, queue: :carrierwave
-  # c.backend :resque, queue: :carrierwave
-  c.backend :sidekiq, queue: :carrierwave
+  c.backend :resque, queue: :carrierwave
```
Comment (galacto) [inline config/initializers/carrierwave_backgrounder.rb]: was this change missed in resque migration?

Comment (galacto) [inline config/initializers/carrierwave_backgrounder.rb]: otherwise looks good to me.

Comment (chcholman) [inline config/initializers/carrierwave_backgrounder.rb]: Yup, it was!

Comment (chcholman) [inline config/initializers/carrierwave_backgrounder.rb]: Thanks @galacto!

**Pull request merged**